panel/src/components/Forms/Blocks/Blocks.vue

```diff
@@ -238,8 +287,73 @@ export default {
     confirmToRemoveSelected() {
       this.$refs.removeSelected.open();
     },
+    copy(e) {
+      // don't copy when the drawer is open
+      if (this.isEditing === true) {
+        return false;
+      }
+
+      // don't copy when there are no blocks yet
+      if (this.blocks.length === 0) {
+        return false;
+      }
+
+      // don't copy when nothing is selected
+      if (this.selectedOrBatched.length === 0) {
+        return false;
+      }
+
+      // don't copy if an input is focused
+      if (this.isInputEvent(e) === true) {
+        return false;
+      }
+
+      let blocks = [];
+
+      this.selectedOrBatched.forEach(id => {
+        const block = this.find(id);
+        if (block) {
+          blocks.push(block);
+        }
+      });
+
+      // don't copy if no blocks could be found
+      if (blocks.length === 0) {
+        return false;
+      }
+
+      const json = JSON.stringify(blocks, null, 2);
```

Comment:
I think moving the actual clipboard logic to a helper would be good, see https://github.com/getkirby/kirby/tree/feature/clipboard-helper

And also adding support for iOS devices (see branch)